cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -1840,6 +1844,47 @@ def setUpClass(cls):
             zip_archive.write(data.read())
         cls._image_sizes[filename] = img_sizes
 
+        filename = "test_pointcloud_pcd.zip"
+        path = os.path.join(os.path.dirname(__file__), 'assets', 'test_pointcloud_pcd.zip')
+        image_sizes = []
+        # container = av.open(path, 'r')
+        zip_file = zipfile.ZipFile(path)
+        for info in zip_file.namelist():
+            if info.rsplit(".", maxsplit=1)[-1] == "pcd":
+                with zip_file.open(info, "r") as file:
+                    data = ValidateDimension.get_pcd_properties(file)
+                    image_sizes.append((int(data["WIDTH"]), int(data["HEIGHT"])))
+        cls._image_sizes[filename] = image_sizes
+
+        filename = "test_velodyne_points.zip"
+        path = os.path.join(os.path.dirname(__file__), 'assets', 'test_velodyne_points.zip')
+        image_sizes = []
+        # create zip instance
+
+        zip_file = zipfile.ZipFile(path, mode='a')
+
+        source_path = []
+        root_path = os.path.abspath(os.path.split(path)[0])
+
+        for info in zip_file.namelist():
+            if info.rsplit(".", maxsplit=1)[-1] == "bin":
```

Comment:
probably it is better to use os.path.splitext here.

Reply:
Made changes accordingly.